graphbolt/src/csc_sampling_graph.cc

```diff
@@ -5,6 +5,7 @@
  */
 
 #include <graphbolt/csc_sampling_graph.h>
+#include <graphbolt/random.h>
```

Comment:
avoid including header without using it.

Reply:
Get, I just test it whether work.